### PR TITLE
Rpm handler syft package.

### PR DESCRIPTION
**Pull request by Toure**
**What this PR does / why we need it**:
Created a rpm handler for anchore-engine to interface with syft, this will replace a portion of the current 10_packages_list module with a call into syft.   https://github.com/anchore/anchore-engine/issues/682

### CODE REVIEW

File: anchore_engine/analyzers/syft/handlers/rpm.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import re
+
+from anchore_engine.analyzers.utils import dig
+
+
+def handler(findings, artifact):
+    """
+    Handler function to map syft results for an alpine package type into the engine "raw" document format.
+    """
+    # _all_package_files(findings, artifact)
```
Comment (alfredodeza) [inline anchore_engine/analyzers/syft/handlers/rpm.py:10]: looks like this needs to get removed?

File: tests/functional/clients/standalone/package_list/test_centos.py [hunk 2]
```diff
@@ -23,5 +23,6 @@ def test_pkgs_all(self, analyzed_data, pkg, version):
     def test_pkgs_allinfo(self, analyzed_data, pkg, metadata):
         result = analyzed_data("rpm")
         pkgs = result['image']['imagedata']['analysis_report']['package_list']['pkgs.allinfo']['base']
-        loaded = json.loads(pkgs.get(pkg, '{}'))
+        # import ipdb; ipdb.set_trace()
```
Comment (alfredodeza) [inline tests/functional/clients/standalone/package_list/test_centos.py:26]: this needs to get removed

File: tests/functional/clients/standalone/package_list/test_centos.py [hunk 3]
```diff
@@ -7,11 +7,11 @@
 
 class TestRpm:
 
-    @pytest.mark.parametrize('path', path_params(centos.pkgfiles_all))
-    def test_pkgfiles_all(self, analyzed_data, path): #, path):
-        result = analyzed_data("rpm")
-        pkgs = result['image']['imagedata']['analysis_report']['package_list']['pkgfiles.all']['base']
-        assert pkgs.get(path) == 'RPMFILE'
+    # @pytest.mark.parametrize('path', path_params(centos.pkgfiles_all))
```
Comment (alfredodeza) [inline tests/functional/clients/standalone/package_list/test_centos.py:10]: are you meant to comment this out? or was this temporary only?

**Review by alfredodeza — CHANGES_REQUESTED** [review]

File: tests/functional/clients/standalone/package_list/test_centos.py (hunk 2 above)
Comment (Toure) [reply tests/functional/clients/standalone/package_list/test_centos.py:26]: Thanks for catching that.

File: tests/functional/clients/standalone/package_list/test_centos.py [hunk 4]
```diff
@@ -23,5 +23,5 @@ def test_pkgs_all(self, analyzed_data, pkg, version):
     def test_pkgs_allinfo(self, analyzed_data, pkg, metadata):
         result = analyzed_data("rpm")
         pkgs = result['image']['imagedata']['analysis_report']['package_list']['pkgs.allinfo']['base']
-        loaded = json.loads(pkgs.get(pkg, '{}'))
+        loaded = pkgs.get(pkg, '{}')
```
Comment (wagoodman) [inline tests/functional/clients/standalone/package_list/test_centos.py:26]: `'{}'` --> `{}`

File: tests/functional/clients/standalone/package_list/test_centos.py (hunk 3 above)
Comment (alfredodeza) [reply tests/functional/clients/standalone/package_list/test_centos.py:7]: why can't this be kept uncommented bith with a `skipif` ? The TODO message could be the `skipif` reason

**Review by Toure — COMMENTED** [review]
Added the skip decorator to the test.

**Review by alfredodeza — APPROVED** [review]

**Pull request merged**